applications/nrf_desktop/src/modules/Kconfig.fast_pair

```diff
@@ -24,6 +24,18 @@ config DESKTOP_FAST_PAIR_LIMIT_NORMAL_PAIRING
 	  Bluetooth pairing when outside of pairing mode (if the used Bluetooth
 	  local identity already has a bonded peer).
 
+config DESKTOP_FAST_PAIR_FACTORY_RESET
+	bool "Enable Fast Pair factory reset"
```

Comment:
Please remove `Enable` as per guide lines:
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#prompt-strings
> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.